nextflow.config

```
@@ -21,6 +21,16 @@ params {
     // fARGene options
     fargene_hmm_model          = 'class_a'
 
+    // DeepARG options
+    deeparg_data                        = null
+    deeparg_model                       = 'ss'
```

```suggestion
    deeparg_model                       = 'SS'
```
See documentation of deepARG